test/widgets/conftest.py

```diff
@@ -74,3 +77,44 @@ class MinimalConf(libqtile.confreader.Config):
         screens = []
 
     return MinimalConf
+
+
+@pytest.fixture(scope='function')
+def dbus(monkeypatch):
+    # for Github CI, dbus-launch is provided by "dbus-x11" package
+    launcher = shutil.which("dbus-launch")
+
+    # If dbus-launch can't be found then tests will fail so we
+    # need to skip
+    if launcher is None:
+        pytest.skip("dbus-launch must be installed")
```

Comment:
Added skip: Skip any test that uses this fixture if `dbus-launch` is unavailable.